### PR TITLE
 rounded form elements and buttons for better UX

### PR DESCRIPTION
**Pull request by ravinduramesh**
Checklist:

- [ ] Correct branch: master for new features; 3.2.x for fixes
- [ ] Tests pass
- [ ] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [ ] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.2.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-12345

### CODE REVIEW

File: phpBB/styles/prosilver/theme/buttons.css [hunk 1]
```diff
@@ -20,6 +20,7 @@
 	-moz-user-select: none;
 	-ms-user-select: none;
 	user-select: none;
+	moz-border-radius: 4px; -webkit-border-radius: 4px; border-radius: 4px;
```
Comment (hanakin) [inline phpBB/styles/prosilver/theme/buttons.css]: This is redundant code

Comment (hanakin) [reply phpBB/styles/prosilver/theme/buttons.css]: also,
firstly when submitting prs please follow the guidelines for commits https://area51.phpbb.com/docs/dev/3.2.x/development/git.html
second please be sure any changes adhere to our coding standards for 3.3 which are currently in draft and can be found here: https://github.com/phpbb/documentation/pull/128
also, our supported browsers are listed here: https://github.com/phpbb/phpbb/blob/master/gulpfile.js#L23

Comment (ravinduramesh) [reply phpBB/styles/prosilver/theme/buttons.css]: Thanks for the guiding. Will check them out.

Comment (hanakin) [reply phpBB/styles/prosilver/theme/buttons.css]: also you have no ticket for this https://tracker.phpbb.com/secure/Dashboard.jspa

**Comment by marc1706** [comment]
Can you provide screenshots of the changes? We'd like to see screenshots of the before and after for your changes.

**Comment by hanakin** [comment]
closing due to inactivy for almost a year

**Pull request closed without merging**